server/plugin/client.go

```diff
@@ -329,3 +330,27 @@ func (c *client) SearchFilterValuesInServiceNow(searchTerm, limit, offset, reque
 
 	return assignmentGroups.Result, statusCode, nil
 }
+
+func (c *client) GetRecordsFromServiceNow(recordType, assignmentGroupID, serviceID string) ([]*serializer.ServiceNowRecord, int, error) {
+	query := "active=true"
```

Comment:
should we create a constant for this

Reply:
I don't think it is required as it is used here only